pkg/newstorage/newstorage.go

```diff
@@ -8,6 +8,12 @@ package newstorage
 
 import "errors"
 
+// BulkOperationError represents one or more errors that occurred during a bulk operation.
+type BulkOperationError interface {
```

Comment:
also this pattern is more generally the multierror.

Reply:
Changed to MultiError.